great_expectations/validator/v1_validator.py

```diff
@@ -48,7 +48,9 @@ def __init__(
         self.result_format = result_format
 
     def validate_expectation(
-        self, expectation: Expectation
+        self,
+        expectation: Expectation,
+        evaluation_parameters: Optional[dict[str, Any]] = None,
```

Comment:
Nit - I think just `dict` is sufficient here instead of `[str, Any]`

Reply:
I like being explicit about the key type.

Reply:
I figure at least giving the key a type gives us a bit more to go on here, but I'm fine to remove it. I could have sworn I saw a PR in one of our repos to require adding the type params.